scripts/lib/spreadsheet-standardizer.ts

```diff
@@ -61,38 +100,30 @@ export class SpreadsheetStandardizer {
     const authorityName = createAuthorityName(state, record.authority_name);
     const output: Record<string, string | number | object | boolean> = {
       id: record.id,
-      authority_type: coerceToStandardValue(
-        record.authority_level,
-        getEnumValues(Object.values(AuthorityType)),
-      ),
+      authority_type: record.authority_type,
       authority: authorityName,
-      type: coerceToStandardValue(
-        record.rebate_type.split(',')[0],
-        getEnumValues(Object.values(PaymentMethod)),
-        VALUE_MAPPINGS.payment_methods,
-      ),
-      payment_methods: findPaymentMethods(record.rebate_type),
-      item: coerceToStandardValue(
-        record.technology,
-        new Set(Object.keys(ITEMS_SCHEMA)),
-        VALUE_MAPPINGS.item,
-      ),
+      type: record.payment_methods.split(',')[0],
+      payment_methods: record.payment_methods.split(',').map(x => x.trim()),
+      item: record.item,
       program: createProgramName(
         state,
         record.authority_name,
         record.program_title,
       ),
-      amount: createAmount(record),
-      owner_status: findOwnerStatus(record.owner_status),
+      amount: this.createAmount(record),
+      owner_status: record.owner_status.split(',').map(x => x.trim()),
```

Comment:
What does this end up with in the "Both" case? we split on the "homeowner, renter" and then trip blankspaces -- is the end result an array with both? Doesn't this need to transform again into a string? 

Sorry if this is a silly typescript syntax misreading.

Reply:
<insert thumbs up here>